histomicstk/segmentation/label/passInf.h

```diff
@@ -0,0 +1,5 @@
+#ifndef INFINITY
+
+#define INFINITY 0
+
+#endif
```

Comment:
I don't see a directly way to define it on .pyx, but I think it's possible to define it on isbfcpp.h. I'll try it.